Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.9-slim
+
+# install dependencies for building oppai_ng
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && apt install -y cmake build-essential
+
+COPY oppai_ng/ /oppai_ng/
+WORKDIR oppai_ng/
+RUN cmake . -DPYTHON_EXECUTABLE=/usr/local/bin/python -G "Unix Makefiles" && make
+
+# install python dependencies
+WORKDIR /
+COPY requirements.txt ./
+RUN pip install -r requirements.txt
+
+# copy the source code in last, so that it doesn't
+# repeat the previous steps for each change
+COPY . .
+
+ENTRYPOINT ["python3"]
+CMD ["main.py"]
```

Comment:
we should run scripts/start.sh here rather than main.py directly

we should use https://github.com/vishnubob/wait-for-it for connected service healthchecks so we start everything in order

Reply:
+1 on that